limitador/src/limit.rs

```diff
@@ -307,6 +311,7 @@ where
 
 impl Limit {
     pub fn new<N: Into<Namespace>, T: TryInto<Condition>>(
+        id: Option<String>,
```

Comment:
Note to mostly myself here, as I'll probably pick this up, but... 
Let's add a `Limit::with_id(String, ..)` or something, and keep `::new` as is, implying the `None`. That would be less of change across all the files, and make I think for a cleaner API

Reply:
Good idea. ❤️ in flight wifi.